rules/windows/process_creation/win_dll_sideload_xwizard.yml

```diff
@@ -0,0 +1,24 @@
+title: Xwizard DLL Sideloading
+id: 193d5ccd-6f59-40c6-b5b0-8e32d5ddd3d1
+status: experimental
+description: Detects the execution of Xwizard tool from the non-default directory which can be used to sideload a custom xwizards.dll
+references:
+    - https://lolbas-project.github.io/lolbas/Binaries/Xwizard/
+    - http://www.hexacorn.com/blog/2017/07/31/the-wizard-of-x-oppa-plugx-style/
+author: Christian Burkard 
+date: 2021/09/20
+tags:
+    - attack.defense_evasion
+    - attack.t1574.002
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\xwizard.exe'
+    filter:
+        Image|startswith: '?:\Windows\System32\'
```

Comment:
HI,
This will search the exact string `?:\Windows\System32\` so will never match 'c:\' or 'd:\' ...